src/devices/develco.ts

```diff
@@ -446,7 +462,9 @@ const definitions: Definition[] = [
             await develco.configure.read_sw_hw_version(device, logger);
         },
         exposes: [e.power(), e.energy(), e.current(), e.voltage(), e.current_phase_b(), e.voltage_phase_b(), e.current_phase_c(),
-            e.voltage_phase_c()],
+            e.voltage_phase_c(),
+            e.numeric('total_active_power', ea.STATE).withUnit('W').withDescription('Total active power'),
```

Comment:
What's the difference with `power`?

Reply:
I have compared the values from power and total_active_power the last 24h, they are the same. 
The main difference is that the total_active_power and the total_reactive_power are both read from haElectricalMeasurement, while power is taken from seMetering. 
To mee it seems more "logical" and correct to take all measurements from haElectricalMeasurement, while the energy (hourly accumulated consumption) is read from the seMetering. Since these are the total values (3 phases, not only one) it makes more "correct" to use total (total_active_power and total_reactive_power), but for the sake of compability I believe 'power' (read from totalActivePower) and 'power_reactive' (read from totalReactivePower) is better. 
@Koenkk , what's your opinion?

Reply:
To be clear, we don't need both power and total_active_power, since they are the same. I propose to keep 'power", but read from totalActivePower.

Reply:
> To be clear, we don't need both power and total_active_power, since they are the same. I propose to keep 'power", but read from totalActivePower.

Fine with that!